src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Collections/SeqModule.fs

```diff
@@ -19,11 +19,11 @@ Make sure each method works on:
 * Null Seq    (null)
 *)
 
-[<TestFixture>]
-type SeqModule() =
+[<Parallelizable>][<Category "Collections Tests">][<Category "Seq Tests">]
+module SeqModuleTests =
```

Comment:
I personally like the simpler `module` and `let` syntax, but I think we should do this as a seperate PR, as @dsyme has said on a number of occasions that he would prefer minimum changes so as to limit as much scope of review as possible.